easy_my_coop_loan/models/loan.py

```diff
@@ -38,6 +28,7 @@ def _compute_subscribed_amount(self):
     user_id = fields.Many2one('res.users',
                               string="Responsible")
     term_date = fields.Date(string="Term date")
+    loan_term = fields.Float(string="term of the loan")

```

Comment:
So the unit of loan_term is _year_? Might be good to put a help message for it.

Reply:
For now it is... that why it's a float. It has been asked to pass it in monts but i'm not sure it's a good idea